scripts/webpack/js-bundle-factory.js

```diff
@@ -51,6 +51,7 @@ class JsBundleFactory {
     {
       bundleName,
       chunks,
+      extensions = ['.ts', '.js', '.json'],
```

Comment:
Why `.json`?

Reply:
I couldn't find what resolve.extensions defaults to, but it is in the code sample of https://webpack.js.org/configuration/resolve/#resolve-extensions. It didn't seem to hurt. Although we do not have any json files we depend on in the component/foundation/adapter, I don't see why it isn't a possibility.

Reply:
IIRC Node.js can import JSON files so that's probably what inspires it. I'd be inclined to leave json out since I don't think we should ever have a reason to be importing one.